source/_components/alarmdecoder.markdown

```diff
@@ -43,6 +43,11 @@ alarmdecoder:
     02:
       name: 'Front Door'
       type: 'opening'
+    03:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
I do agree with the changes.  To follow with this the rule rfid should also be removed from zone 1 that is not part of my current proposed changes.